scripts/linters/other_files_linter.py

```diff
@@ -319,7 +319,7 @@ def _check_that_workflow_steps_use_merge_action(
         """
         jobs_without_merge = []
         for job, job_dict in workflow_dict['jobs'].items():
-            if MERGE_STEP not in job_dict['steps']:
+            if 'steps' in job_dict and MERGE_STEP not in job_dict['steps']:
```

Comment:
I don't think we should exempt workflows that lack steps from this lint check. Instead, we should correctly parse the job and add the new workflow to the exclusion list since it doesn't need to merge from develop

Reply:
Done.